src/user-guide/writing-workflows/scheduling.rst

```diff
@@ -393,12 +411,40 @@ So, for example, you can write:
 .. code-block:: sub
 
    R1//+P0D  # Run once at the final cycle point
-   R5/P1D    # Run 5 times, every 1 day, ending at the final
-             # cycle point
-   P2W/T00   # Run every 2 weeks ending at 00:00 following
-             # the final cycle point
-   R//T00    # Run every 1 day ending at 00:00 following the
-             # final cycle point
+   R5/P2D    # Run 5 times, every 2 days, ending at the final cycle point
+   P2W/T00   # Run every 2 weeks ending at 00:00 before/at the final cycle point
+   R//T00    # Run every day ending at 00:00 before/at the final cycle point
+
+
+Format 1: ``R[limit?]/[datetime]/[datetime]``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A less common recurrence form is ``R[limit?]/[datetime]/[datetime]``
+(format number 1 in the ISO 8601 standard). This uses the difference between
+the first datetime and the second datetime to set the recurrence interval.
+The first datetime is the start point. For example,
+``R3/2020-07-10/2020-07-15`` means
+
```

Comment:
I think that you should write this in plain English too:

"Run three times / starting with the 10th July 2020 / and repeating on the 15th (and using the interval between the 10th and 15th of 5 days to calculate further recurrences)"